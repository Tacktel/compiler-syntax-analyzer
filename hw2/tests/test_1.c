int func(int a) {
    a = a * 1;
    return 0; 
}