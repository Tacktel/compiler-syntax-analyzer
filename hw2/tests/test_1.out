INT:int
WSPACE
ID:func
LPAREN:(
INT:int
WSPACE
ID:a
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
ID:a
WSPACE
ASSIGN:=
WSPACE
ID:a
WSPACE
OP:*
WSPACE
INTVAL:1
TERM:;
WSPACE
RETURN:return
WSPACE
INTVAL:0
TERM:;
WSPACE
RSCOPE:}
